src/crccombine.h

```diff
@@ -0,0 +1,17 @@
+
+#include <stdint.h>
+
+
+#define USE_STATIC_COMBINE_CACHE 1
+// mask types
+typedef unsigned long long v2uq __attribute__ ((vector_size (16)));
+
+// final implementation
+uint64_t gf2_matrix_times_vec2(uint64_t *mat, uint64_t vec);
+
+
+#if USE_STATIC_COMBINE_CACHE
```

Comment:
Do we expect people to disable this? Otherwise this is always true and we can just remove it.

Reply:
removed